### ISSUE TITLE
Switch to POST if the request URI on a GET is too long

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#96]
```
Switch to POST if the request URI on a GET is too long. Form encode the query 
parameters into the body and set the header X-HTTP-Method-Override: GET.
```

Original issue reported on code.google.com by `jcgregorio@google.com` on 6 Feb 2012 at 3:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 6 Feb 2012 at 3:48
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 7 Feb 2012 at 7:37
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Is their a workaround for this? Or do I have to make my request manually?
```

Original comment by `philipbj...@gmail.com` on 6 Jul 2012 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
CL out for review:  http://codereview.appspot.com/6454092/
```

Original comment by `jcgregorio@google.com` on 3 Aug 2012 at 3:23

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7d1515ca91f0.
```

Original comment by `jcgregorio@google.com` on 3 Aug 2012 at 4:48
- Changed state: **Fixed**
